pages/common/rbw.md

```diff
@@ -0,0 +1,36 @@
+# rbw
+
+> Unofficial Bitwarden-compatible password manager.
+> More information: <https://github.com/doy/rbw#configuration>.
```

Comment:
```suggestion
> More information: <https://github.com/doy/rbw>.
```

Reply:
This was explicitly requested in an earlier review: https://github.com/tldr-pages/tldr/pull/17937#discussion_r2314324662